### DISCUSSION TITLE
Stuck on walk mode.

### DISCUSSION THREAD

**Discussion by QuenisKleist** [danielnilsson9/bbs-fw#212] [Q&A]
Installed with the main need to disable the speed sensor. That part worked. 

However, it seems to be stuck in walk mode. I've tried disabling the mode entirely but when connected in any way, the bafang keeps a walkin'. This happened immediately after flashing the hex file up. I tried twice to make sure but the same thing happened each time. It'll stop walking only if I have it set so that the motor is disabled in one of the levels. I have a bbshd. My display only allows five levels. The controller was bought within the last eight months. Any ideas on how to stop the walk?

**Comment by S-CODE-pl** [comment]
if you have throttle try to increase min throttle voltage

**Reply by QuenisKleist** [reply]
Oh interesting. One of the things I tried yesterday was to decrease the min throttle voltage but it kept throwing me 05h errors. I'll try to increase it instead.  

Edit: That fixed it! Upped the min throttle voltage to 1000 and the issue went away. You're a godsend.
